builds/fabric8-che/plugins/keycloak-plugin-token-provider/src/main/java/com/redhat/che/keycloak/token/provider/contoller/TokenController.java

```diff
@@ -105,10 +109,20 @@ public Response getOpenShiftToken(@HeaderParam(HttpHeaders.AUTHORIZATION) String
         try {
             validator.validate(keycloakToken);
             token = tokenProvider.obtainOsoToken(keycloakToken);
-        } catch (KeycloakException e) {
+        } catch (Exception e) {
             return Response.status(Response.Status.BAD_REQUEST).entity(e.getMessage()).build();
         }
         return Response.ok(token).build();
     }
 
+    @GET
+    @Path("/user")
+    public Response getUserMatches(@HeaderParam(HttpHeaders.AUTHORIZATION) String keycloakToken) {
```

Comment:
well, there no Javadoc on the other methods as well, maybe we could create an issue so that the respective authors of those various methods (some of which come from previous PRs) would add Javadoc all at once ?